README.md

```diff
@@ -6,12 +6,14 @@ A sample Windows application that connects to an Aras instance.
 
 Release | Notes
 --------|--------
+[v1.0.1](https://github.com/ArasLabs/innovator-client-example/releases/tag/v1.0.1) | Updated Install steps 
 [v1](https://github.com/ArasLabs/innovator-client-example/releases/tag/v1) | Migrated from the old Aras Community Projects site and updated.
 
 #### Supported Aras Versions
 
 Project | Aras
 --------|------
+[v1.0.1](https://github.com/ArasLabs/innovator-client-example/releases/tag/v1.0.1) | Tested on 11.0 SP15, SP12
```

Comment:
This project still supports 10.0 SPx and 11.0 SPx. You can include the tested versions in the release notes above.
```suggestion
[v1.0.1](https://github.com/ArasLabs/innovator-client-example/releases/tag/v1.0.1) | 10.0 SPx, 11.0 SPx
```